src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -13,10 +13,9 @@
 
 namespace Microsoft.Oryx.BuildScriptGenerator.Node
 {
+    // TODO: is this used anywhere?
```

Comment:
can you please remove this from your view? I'm working on this as part of my changes and want to avoid conflict :)